mitmproxy/contentviews/multipart.py

```diff
@@ -21,4 +21,7 @@ def __call__(self, data: bytes, content_type: Optional[str] = None, **metadata):
             return "Multipart form", self._format(v)
 
     def render_priority(self, data: bytes, *, content_type: Optional[str] = None, **metadata) -> float:
-        return float(content_type == "multipart/form-data")
+        if content_type and content_type.startswith("multipart/form-data"):
```

Comment:
see above - this is only used to select the correct view, we don't need to handle the boundary information here.